src/pykiso/lib/connectors/cc_pcan_can.py

```diff
@@ -379,6 +379,7 @@ def _cc_receive(
             log.exception(f"encountered error while receiving message via {self}")
             return {"msg": None}
 
+    # TODO: refactor to use trc reader when new version of python can is release
```

Comment:
Please create a ticket to track this (very well described) and remove it from here.

Reply:
ticket internally created

Reply:
added issue "use official trc reader"